yabt/builders/cpp.py

```diff
@@ -368,7 +371,9 @@ def cpp_gtest_tester(build_context, target):
     test_cmd.extend(target.props.test_flags)
     build_context.run_in_buildenv(
         # TODO: target.props.in_testenv,
-        target.props.in_buildenv, test_cmd, target.props.test_env)
+        target.props.in_buildenv, test_cmd, target.props.test_env,
+        None, True, target.props.runtime
```

Comment:
better use keyword arg, and let the func signature take care of default values for other args

```suggestion
        runtime=target.props.runtime
```